### DISCUSSION TITLE
How to upgrade Airflow v2 scheduler performance.

### DISCUSSION THREAD

**Discussion by socar-grab** [apache/airflow#16210] [Q&A]
Hi there! I'm now upgrading to 2.1.0 from 1.10.14

I read the document and saw some codes about scheduling.
![image](https://user-images.githubusercontent.com/83575607/120407363-f8606400-c387-11eb-9dda-569229133a9c.png)

I understand 'parsing processes' of scheduler are associated with dag serialization only and 'scheduler job loop' works from creating dag runs and task instances to sending tasks to executor's queue. 

[questions]
1. I think 'parsing_processes' parameter from airflow.cfg  is related to parsing dag and save into db not overall scheduling job.  It is right?

2. If I want to make scheduler bear the many dagruns execution in same time, which configurations should I touch?

Thank you!

**Comment by potiuk** [comment]
[1] yes
[2] I think there is no single answer - depends on what DB/workers/dag size/dag num you have. I think the best "summary" of all the tunables are here:  https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html#scheduler-tuneables
